docs/setup/prerequisites.md

```diff
@@ -143,7 +143,7 @@ Core:
   Download and install it with:
 
   ```
-  export GOFUMPT_VERSION=abc0db2c416aca0f60ea33c23c76665f6e7ba0b6
+  export GOFUMPT_VERSION=v0.1.1
   GO111MODULE=on ${OASIS_GO:-go} get mvdan.cc/gofumpt@${GOFUMPT_VERSION}
   GO111MODULE=on ${OASIS_GO:-go} get mvdan.cc/gofumpt/gofumports@${GOFUMPT_VERSION}
```

Comment:
I think we could just replace `gofumports` with `goimports`:

```suggestion
  GO111MODULE=on ${OASIS_GO:-go} get mvdan.cc/gofumpt@v0.1.1
  ${OASIS_GO:-go} get golang.org/x/tools/cmd/goimports
```

And change the `fmt` target in `go/Makefile` to:
```
# Format code.
fmt:
	@$(ECHO) "$(CYAN)*** Running Go formatters...$(OFF)"
	@gofumpt -s -w .
	@goimports -w -local github.com/oasisprotocol/oasis-core .
```